db/models.py

```diff
@@ -50,3 +57,58 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
```

Comment:
The `__str__` method in `MovieSession` provides a clear representation of the session, combining movie title and show time.